app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
```

Comment:
Replace this import with the following
```suggestion
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```
There is no need to duplicate `from app.errors import`